packages/material-ui-lab/src/TimelineDot/TimelineDot.test.js

```diff
@@ -1,20 +1,18 @@
 import * as React from 'react';
-import { getClasses, createMount, describeConformance } from 'test/utils';
-import TimelineDot from './TimelineDot';
+import { createClientRender, createMount, describeConformanceV5 } from 'test/utils';
+import TimelineDot, { timelineDotClasses as classes } from '@material-ui/lab/TimelineDot';
 
 describe('<TimelineDot />', () => {
+  const render = createClientRender();
   const mount = createMount();
-  let classes;
 
-  before(() => {
-    classes = getClasses(<TimelineDot />);
-  });
-
-  describeConformance(<TimelineDot />, () => ({
+  describeConformanceV5(<TimelineDot />, () => ({
     classes,
     inheritComponent: 'span',
+    render,
     mount,
+    muiName: 'MuiTimelineDot',
     refInstanceof: window.HTMLSpanElement,
-    skip: ['componentProp'],
+    skip: ['componentProp', 'componentsProp', 'themeVariants'],
```

Comment:
Let's add the `themeVariants` tests, as this component supports more props. Was there an issue as to why this was skipped?